src/documentation/sdks/nexmark.md

```diff
@@ -417,124 +429,192 @@ Yet to come
 
 ### Running SMOKE suite on the DirectRunner (local)
 
+The DirectRunner is default, so it is not required to pass `-Pnexmark.runner`.
+Here we do it for maximum clarity.
+
+The direct runner does not have separate batch and streaming modes, but the
+Nexmark launch does.
+
+These parameters leave on many of the DirectRunner's extra safety checks so the
+SMOKE suite can make sure there is nothing broken in the Nexmark suite.
+
 Batch Mode:
 
-    mvn exec:java -Dexec.mainClass=org.apache.beam.sdk.nexmark.Main -Pdirect-runner -Dexec.args="--runner=DirectRunner --suite=SMOKE --streaming=false --manageResources=false --monitorJobs=true --enforceEncodability=true --enforceImmutability=true"
+    ./gradlew :beam-sdks-java-nexmark:run \
+        -Pnexmark.runner=":beam-runners-direct-java" \
+        -Pnexmark.args="
+            --runner=DirectRunner
+            --streaming=false
+            --suite=SMOKE
+            --manageResources=false
+            --monitorJobs=true
+            --enforceEncodability=true
+            --enforceImmutability=true"
 
 Streaming Mode:
 
-    mvn exec:java -Dexec.mainClass=org.apache.beam.sdk.nexmark.Main -Pdirect-runner -Dexec.args="--runner=DirectRunner --suite=SMOKE --streaming=true --manageResources=false --monitorJobs=true --enforceEncodability=true --enforceImmutability=true"
-
+    ./gradlew :beam-sdks-java-nexmark:run \
+        -Pnexmark.runner=":beam-runners-direct-java" \
+        -Pnexmark.args="
+            --runner=DirectRunner
+            --streaming=true
+            --suite=SMOKE
+            --manageResources=false
+            --monitorJobs=true
+            --enforceEncodability=true
+            --enforceImmutability=true"
 
 ### Running SMOKE suite on the SparkRunner (local)
 
+The SparkRunner is special-cased in the Nexmark gradle launch. The task will
+provide the version of Spark that the SparkRunner is built against, and
+configure logging.
```

Comment:
Sorry but I don't understand this comment: owing to what's bellow, the version is automatically provided and no parameter needs to be set by the user. So why not just remove this comment if it is not targeted to the user of nexmark?
Is there something I missed?

Reply:
This is a comment about the hardcoding above. The nexmark build tweaks the dependencies based on special needs of the SparkRunner instead of being just a generic suite of pipelines. I will delete the comment here since it "just works" for users copying and pasting from here.